README.md

```diff
@@ -1,46 +1,123 @@
 # Rails URL Helpers
 
-## Objectives
+Since Rails by design was meant to be flexible, the result is that there are typically a number of ways to accomplish the same features. Routes are a great example for how this principle operates in a Rails app. As you have already seen in our Capybara feature specs, we have the ability to hardcode paths or to use view helper methods. In this section we will extend this knowledge and walk though both options and how they can be used in the implementation code itself.
 
-1. Describe why route helpers are useful (relying on abstractions such as methods as opposed to literring our application with URL data)
-2. Use rake routes to find the name of a route
-3. Predict route names based on rails routing conventions
-4. Use link_to to generate an <a>
-5. Use _path and _url helpers to generate URLs
-6. Name routes with :as option
-7. Generate URLs with route helpers that require variables.
 
+## Paths vs Route Helpers
 
-## Notes
+So why would we want to use route helper methods as opposed to hard coding paths into the application? There are a number of reasons, below are a few of the key rationales:
```

Comment:
Because they don't know the term route helper methods, we need show them the problem with using urls, not just tell them about it. The easiest one to show is your third bullet point of their dynamicism. So I would show them how you set up a route in the routes to `/posts` and then put a link to "/posts" then ask them, what if you change your url now to "/entries"? What if we had a method that would just look at our routes file and put in the correct URL? Thankfully, Rails gives us route helper methods to solve this problem.